### DISCUSSION TITLE
Changing location of Options page in BEX

### DISCUSSION THREAD

**Discussion by thdoan** [quasarframework/quasar#12014] [CLI - BEX mode]
I have this route:

```
  {
    path: '/options',
    component: () => import('layouts/MainLayout.vue'),
    children: [
      {
        path: '',
        component: () => import('pages/Options.vue')
      }
    ]
  },
```

This translates to chrome-extension://ekkndmcdidlhjpfbolniajndhleaobpg/www/index.html#/options

How can I configure my route so that the Options page URI looks like below?

chrome-extension://ekkndmcdidlhjpfbolniajndhleaobpg/options.html#

**Comment by MilosPaunovic** [comment]
Have you set up your `manifest.json` as in the docs?
[https://quasar.dev/quasar-cli/developing-browser-extensions/types-of-bex#dev-tools-options-and-popup](https://quasar.dev/quasar-cli/developing-browser-extensions/types-of-bex#dev-tools-options-and-popup)
